### ISSUE TITLE
Add text language / direction attributes

### ISSUE THREAD

**Issue by r12a** [w3c/i18n-activity#248]
§ https://github.com/w3c/web-share/issues/6
originally at WICG/web-share 6

**Comment by KatyMawhood** [comment]
Since this issue was raised in response to a rework of the note on [json bidi](http://w3c.github.io/i18n-discuss/notes/json-bidi.html) (now complete), I propose to close this issue.

**Comment by aphillips** [comment]
This had the wrong label. It is waiting on future work. Removing close.
